src/components/Tab/Tab.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Tab.css';
+
+export const Tab = ({ tabs, currentTabId }) => (
+  <>
+    <ul
```

Comment:
links to tabs should be rendered inside TabsPage

this component should be renamed to TabContent or should render single `li` of this list